mjml/core/api.py

```diff
@@ -72,7 +71,7 @@ def getContent(self):
         # mjml's "email-templates" repo.
         if self.content is None:
             return ''
-        return self.content.strip()
+        return str(self.content).strip()
```

Comment:
Might not be necessary here to cast to a string, since this should already be one (I think this is a remnant from WIP).